SAM_UI_TOOL/SAM_image_segmentation.py

```diff
@@ -28,13 +27,18 @@ def __init__(self, args, master, images: List[str]) -> None:
             self.buttons, text="Previous", command=self.previous
         )
         self.previous_button.grid(row=0, column=1)
-        self.done_button = ttk.Button(self.buttons, text="Done", command=self.done)
+        self.next_button = ttk.Button(self.buttons, text="Next", command=self.next)
+        self.next_button.grid(row=0, column=4)
+        self.done_button = ttk.Button(
+            self.buttons, text="Done", command=self.done, state="disabled"
+        )
         self.done_button.grid(row=0, column=2)
-        self.exit_button = ttk.Button(self.buttons, text="Exit", command=master.quit)
-        self.exit_button.grid(row=0, column=3)
+        self.exit_button = ttk.Button(self.buttons, text="Exit", command=self.exit)
+        self.exit_button.grid(row=0, column=20)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏column 20? Which one were you trying to use?

Reply:
oops...i wanted this button to be right-most, that's who 20, but i guess 4 should be the correct number